src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -225,31 +244,32 @@
            token-not-supported-in-receiver-networks?]}]
   [rn/view
    (map-indexed (fn [index {:keys [chain-id total-amount type]}]
-                  [rn/view
-                   {:key   (str (if receiver? "to" "from") "-" chain-id)
-                    :style {:margin-top (if (pos? index) 11 7.5)}}
-                   [quo/network-bridge
-                    {:amount        (if (= type :not-available)
-                                      (i18n/label :t/not-available)
-                                      (str total-amount " " token-symbol))
-                     :network       (network-utils/id->network chain-id)
-                     :status        (cond (and (= type :not-available)
-                                               loading-routes?
-                                               token-not-supported-in-receiver-networks?)
-                                          :loading
-                                          (= type :not-available)
-                                          :disabled
-                                          :else type)
-                     :on-press      #(when (not loading-routes?)
-                                       (cond
-                                         (= type :edit)
-                                         (open-preferences)
-                                         on-press (on-press chain-id total-amount)))
-                     :on-long-press #(when (not loading-routes?)
-                                       (cond
-                                         (= type :add)
-                                         (open-preferences)
-                                         on-long-press (on-long-press chain-id)))}]])
+                  (let [status (cond (and (= type :not-available)
+                                          loading-routes?
+                                          token-not-supported-in-receiver-networks?)
+                                     :loading
+                                     (= type :not-available)
+                                     :disabled
+                                     :else type)]
+                    [rn/view
+                     {:key   (str (if receiver? "to" "from") "-" chain-id)
```

Comment:
"to and "from" should be i18n labels, no? perhaps they both should take this chain-id as a param too so they're translatable?

Reply:
Well, it is `:key`, so it is probably unimportant. Moreover, we aren't sure how react will work with translated keys)

Reply:
Good point, I missed that